NitroxModel/DataStructures/GameLogic/VehicleMovementData.cs

```diff
@@ -36,12 +36,20 @@ public class VehicleMovementData
         [ProtoMember(9)]
         public bool AppliedThrottle { get; }
 
+        [ProtoMember(10)]
+        public Vector3 LeftAimTarget { get; }
```

Comment:
How are these members used?  Not thrilled about them being the generic VehicleMovementData packet as this is an extra 2 members for every vehicle movement packet.

Reply:
As discussed lets create a factory for VehicleMovementData.